src/main/java/mate/academy/internetshop/controllers/ProductsViewController.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.service.ProductService;
+
+@WebServlet ("/products")
+public class ProductsViewController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllProductsController extends HttpServlet {
```

Reply:
Initially, it was like this but I decided to name them as view because it's more convenient for me as I understand that there I can view products, can I leave it like this?